src/webgpu/shader/validation/variable_and_const.spec.ts

```diff
@@ -137,12 +137,37 @@ g.test('v_0038')
     const { storageClass, containerType, scalarType } = t.params;
     const type = containerType ? `${containerType}<${scalarType}>` : scalarType;
 
-    const code = `
-      [[location(0)]] var<${storageClass}> a : ${type} = ${type}();
+    var code;
+    if (`${storageClass}` === 'in') {
+      code = `
+        struct MyInputs {
+          [[location(0)]] a : ${type};
+        };
+
+        [[stage(vertex)]]
+        fn main(inputs : MyInputs) {
```

Comment:
The WGSL spec now requires that all vertex shaders return at least `[[builtin(position)]]`.

Reply:
Converted to fragment